jbmc/src/java_bytecode/java_bytecode_parser.cpp

```diff
@@ -1527,50 +1525,46 @@ void java_bytecode_parsert::relement_value_pair(
       UNUSED_u2(type_name_index);
       UNUSED_u2(const_name_index);
       // todo: enum
+      return exprt();
```

Comment:
Would it make sense to return an `optionalt<exprt>` to make it clear it is not useful (and in callsite, only store if it has a value)

Reply:
Probably not, as this should be implemented properly instead in a subsequent PR.